internal/services/appservice/linux_web_app_slot_resource_test.go

```diff
@@ -2972,10 +2990,12 @@ resource "azurerm_service_plan" "test" {
 }
 
 resource "azurerm_linux_web_app" "test" {
-  name                = "acctestWA-%[1]d"
-  location            = azurerm_resource_group.test.location
-  resource_group_name = azurerm_resource_group.test.name
-  service_plan_id     = azurerm_service_plan.test.id
+  name                                           = "acctestWA-%[1]d"
+  location                                       = azurerm_resource_group.test.location
+  resource_group_name                            = azurerm_resource_group.test.name
+  service_plan_id                                = azurerm_service_plan.test.id
+  ftp_publish_basic_authentication_enabled       = false
+  webdeploy_publish_basic_authentication_enabled = false
```

Comment:
Since this is a base template we shouldn't be setting any `Optional` properties in here
```suggestion
```